packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -497,8 +497,9 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     return [maxVisitStartDatetime, minVisitStopDatetime];
   }, [visitToEdit]);
 
-  const visitStartDate = getValues('visitStartDate');
-  minVisitStopDatetime = minVisitStopDatetime ?? Date.parse(visitStartDate.toLocaleString());
+  const visitStartDate = getValues('visitStartDate') || new Date();
```

Comment:
Why are we optionally calling `new Date()` here? It looks like its already getting called conditionally higher up when initializing `defaultValues` [higher up](https://github.com/openmrs/openmrs-esm-patient-chart/blob/ac5f146ac02036214ad81240acc179817799aac4/packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx#L141).